docs/contracts.rst

```diff
@@ -1051,13 +1086,13 @@ but they can be used as base contracts::
         function utterance() public returns (bytes32) { return "miaow"; }
     }
 
+Abstract contracts can implement most of a contract, but leave some methods abstract, facilitating patterns like `Template method <https://en.wikipedia.org/wiki/Template_method_pattern>`_ and removing code duplication.
```

Comment:
Perhaps "leave some methods unimplemented" is less confusing since there is no way to mark a method "abstract".